modules/navier_stokes/test/tests/finite_volume/ins/action/errors/tests

```diff
@@ -275,7 +275,8 @@
     type = 'RunException'
     input = 2d-rc-error-action.i
     requirement = 'The system shall throw an error if the user defines a non-existing block for thermal conductivity'
-    cli_args = "Modules/NavierStokesFV/thermal_conductivity_blocks='1; 3' "
+    cli_args = "Modules/NavierStokesFV/block='1 2' "
```

Comment:
To trigger an error. Because the variables will be `ANY_BLOCK_ID` and I am trying to throw an error if the supplied velocity and pressure have a different block restriction. There is apparently a difference between the two.

Reply:
So you do still want this line change?

Reply:
I had to keep both lines.